include/mbedtls/compat-1.3.h

```diff
@@ -1377,7 +1377,6 @@
 #define SSL_ANTI_REPLAY_ENABLED MBEDTLS_SSL_ANTI_REPLAY_ENABLED
 #define SSL_ARC4_DISABLED MBEDTLS_SSL_ARC4_DISABLED
 #define SSL_ARC4_ENABLED MBEDTLS_SSL_ARC4_ENABLED
-#define SSL_BUFFER_LEN MBEDTLS_SSL_BUFFER_LEN
```

Comment:
Unfortunately, `SSL_BUFFER_LEN` was a (semi-)public API PolarSSL in 1.3, so we can't remove it from the compatibility translation. It needs to be implemented in a way that works when neither `MBEDTLS_SSL_IN_CONTENT_LEN` nor `MBEDTLS_SSL_OUT_CONTENT_LEN` are defined (since these did not exist in 1.3).

Reply:
Have implemented as the minimum of `MBEDTL_SSL_IN_BUFFER_LEN` & `MBEDTLS_OUT_BUFFER_LEN` (which probably have equal values in any 1.3-compatible setup, but I figure it can't hurt to be certain in case of some mixed codebase.